src/java.base/share/classes/java/text/Collator.java

```diff
@@ -114,10 +113,22 @@
  * against other {@code CollationKey}s. A {@code CollationKey} is
  * created by a {@code Collator} object for a given {@code String}.
  * <br>
- * <strong>Note:</strong> {@code CollationKey}s from different
+ * @apiNote {@code CollationKey}s from different
  * {@code Collator}s can not be compared. See the class description
  * for {@link CollationKey}
  * for an example using {@code CollationKey}s.
+ * @implNote Implementations of {@code Collator} class may produce
+ * different instances based on the "{@code co}"
+ * <a href="https://www.unicode.org/reports/tr35/#UnicodeCollationIdentifier">
+ * Unicode collation identifier</a> in the specified {@code Locale}.
+ * For example:
+ * <blockquote>
+ * {@snippet lang = java:
+ * Collator.getInstance(Locale.forLanguageTag("sv-u-co-trad"));
+ * }
+ * </blockquote>
+ * may return a {@code Collator} instance with the traditional sorting, which
+ * gives 'v' and 'w' the same sorting order.
```

Comment:
The overall intent and wording looks okay but I wonder if it should be in the getInstance methods instead. Also wondering whether impNote is right because it's both informational for developers using the API and useful to implementors.

Reply:
Thanks. Will move it out of the class description into each `getInstance()` method, changing @implNote to @apiNote